sandbox/scripts/delete_stack.sh

```diff
@@ -4,34 +4,56 @@ source /etc/environment
 
 AWS_SECURITY_GROUP=$(curl http://169.254.169.254/latest/meta-data/security-groups)
 
-#DELETE tf nodes
-
 NODES=($(aws ec2 describe-instances \
     --filters "Name=tag-value,Values=${AWS_STACK_NAME}*" \
     --filters "Name=instance.group-name,Values=${AWS_SECURITY_GROUP}" \
     --query 'Reservations[*].Instances[*].[InstanceId, Tags[?Key==`Name`].Value | [0]]' \
     --output text | grep -v ${AWS_STACK_NAME}-master-node | awk '{print $1}'))
-#remove all ALBs tagged by clustername
+
+K8S_MASTER=$(aws ec2 describe-instances \
+    --filters "Name=tag-value,Values=${AWS_STACK_NAME}*" \
+    --filters "Name=instance.group-name,Values=${AWS_SECURITY_GROUP}" \
+    --query 'Reservations[*].Instances[*].[PublicDnsName, InstanceId, Tags[?Key==`Name`].Value | [0]]' \
+    --output text | grep aws_control | awk '{print $1}')
+
+K8S_WORKERS=($(aws ec2 describe-instances \
+    --filters "Name=tag-value,Values=${AWS_STACK_NAME}*" \
+    --filters "Name=instance.group-name,Values=${AWS_SECURITY_GROUP}" \
+    --query 'Reservations[*].Instances[*].[PublicDnsName, InstanceId, Tags[?Key==`Name`].Value | [0]]' \
+    --output text | grep aws_compute | awk '{print $1}'))
+
+#DELETE test LB chart
+scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no /tmp/sandbox/templates/*.yaml centos@$K8S_MASTER:~
+ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -t $K8S_MASTER << EOF1
```

Comment:
heredoc not needed here
ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -t $K8S_MASTER sudo helm delete --purge  my-alb